### PR TITLE
Melnychenko header slider

### PR DESCRIPTION
**Pull request by nmelnychenko**

### CODE REVIEW

File: dist/js/q7.js [hunk 1]
```diff
@@ -1 +1,72 @@
...
+new WOW().init();
+
+
+// Header
+
+// Controls
+const header_quattro_control = document.getElementById("header_quattro_control");
+const header_etron_control = document.getElementById("header_etron_control");
+const header_sq7tdi_control = document.getElementById("header_sq7tdi_control");
+
```
Comment (clocklab) [inline dist/js/q7.js]: in future do not use ID to identify elements

File: dist/js/q7.js [hunk 2]
```diff
@@ -1 +1,72 @@
...
+// Items
+const header_slider = document.getElementById("header_slider");
+const header_quattro = document.getElementById("header_quattro");
+const header_etron = document.getElementById("header_etron");
+const header_sq7tdi = document.getElementById("header_sq7tdi");
+
+header_quattro.className += " active";
+header_quattro_control.className += " cls-2";
```
Comment (clocklab) [inline dist/js/q7.js]: Use `classList.add` instead `.className +=`

File: dist/js/q7.js [hunk 3]
```diff
@@ -1 +1,72 @@
...
+
+const removeHideClass = function () {
+    let hidden_item = document.querySelector(".hide");
```
Comment (clocklab) [inline dist/js/q7.js]: use const

File: dist/js/q7.js [hunk 4]
```diff
@@ -1 +1,72 @@
...
+    hidden_item ? hidden_item.classList.remove("hide") : true;
```
Comment (clocklab) [inline dist/js/q7.js]: use structure `hidden_item && hidden_item.classList.remove("hide")` instead ternary operator

File: dist/js/q7.js [hunk 5]
```diff
@@ -1 +1,72 @@
...
+};
+
+const hideActiveItem = function () {
+    let active_item = document.querySelector(".active");
+    let active_control = document.querySelector(".cls-2");
+    active_item.classList.remove("active");
```
Comment (clocklab) [inline dist/js/q7.js]: see below

File: dist/js/q7.js [hunk 6]
```diff
@@ -1 +1,72 @@
...
+    active_control.classList.remove("cls-2");
+};
+
+header_quattro_control.onclick = function () {
+
+    if (!header_quattro_control.classList.contains("cls-2")) {
+        removeHideClass();
+        hideActiveItem();
+        function changeSlide() {
+            header_quattro.className += " active";
```
Comment (clocklab) [inline dist/js/q7.js]: see below

File: src/js/q7.js [hunk 7]
```diff
@@ -1 +1,72 @@
...
+
+const removeHideClass = function () {
+    let hidden_item = document.querySelector(".hide");
+    hidden_item ? hidden_item.classList.remove("hide") : true;
+};
+
+const hideActiveItem = function () {
+    let active_item = document.querySelector(".active");
+    let active_control = document.querySelector(".cls-2");
+    active_item.classList.remove("active");
```
Comment (clocklab) [inline src/js/q7.js]: add empty line after declaration

File: dist/js/q7.js [hunk 8]
```diff
@@ -1 +1,72 @@
...
+// Header
+
+// Controls
+const header_quattro_control = document.getElementById("header_quattro_control");
+const header_etron_control = document.getElementById("header_etron_control");
+const header_sq7tdi_control = document.getElementById("header_sq7tdi_control");
+
+// Items
+const header_slider = document.getElementById("header_slider");
+const header_quattro = document.getElementById("header_quattro");
```
Comment (clocklab) [inline dist/js/q7.js]: do not use semicolon in the  end of line

Comment (nmelnychenko) [inline dist/js/q7.js]: Fixed

File: dist/js/q7.js (hunk 2 above)
Comment (nmelnychenko) [inline dist/js/q7.js]: fixed

File: dist/js/q7.js (hunk 3 above)
Comment (nmelnychenko) [inline dist/js/q7.js]: fixed

File: dist/js/q7.js (hunk 4 above)
Comment (nmelnychenko) [inline dist/js/q7.js]: fixed

File: dist/js/q7.js (hunk 5 above)
Comment (nmelnychenko) [inline dist/js/q7.js]: fixed

File: dist/js/q7.js (hunk 6 above)
Comment (nmelnychenko) [inline dist/js/q7.js]: fixed

File: src/js/q7.js (hunk 7 above)
Comment (nmelnychenko) [inline src/js/q7.js]: fixed

**Pull request merged**